components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3311,7 +3315,7 @@ def definition_body(self):
 <*mut JSObject>::post_barrier((*cache).as_mut_ptr().offset(PrototypeList::Constructor::%(id)s as isize),
                               ptr::null_mut(),
                               interface.get());
-""" % {"id": name, "name": str_to_const_array(name)})
+""" % {"id": name, "name": str_to_byte_slice(name)})
```

Comment:
Dis would require change in https://github.com/servo/servo/blob/1c6b74e1f16f6c92f80410d98608064fa4fbede2/components/script/dom/bindings/interface.rs#L214-L230